lib/ansible/modules/cloud/ovirt/ovirt_template.py

```diff
@@ -486,18 +530,40 @@ def main():
             service=templates_service,
         )
 
+        entity = None
+        if module.params['version'] is not None and module.params['version'].get('number') is not None:
+            entity = find_subversion_template(module, templates_service)
+            # when user puts version number which does not exist
+            if entity is None:
+                raise ValueError(
+                    "Template with name '%s' and version '%s' in cluster '%s' was not found'" % (
+                        module.params['name'],
+                        module.params['version']['number'],
+                        module.params['cluster'],
+                    )
+                )
+
         state = module.params['state']
         if state == 'present':
+            # when user put version name, but not version number it will create template with version
+            force_create = False
```

Comment:
Please write a better comment like:

When user want to create new template subversion, we must make sure template is force created as it already exists, but new version should be created.

Reply:
An maybe put it directly to create call like:

```python
ret = templates_module.create(
  entity=entity,
  # When user want to create new template subversion, we must make sure template is force created as it already exists, but new version should be created.
  force_create=entity is None and module.params['version'] is not None,
  ...
)
```